test/common/include/alpaka/test/acc/Acc.hpp

```diff
@@ -253,17 +224,20 @@ namespace alpaka
                     // size_t is most probably identical to either std::uint64_t or std::uint32_t.
                     // This would lead to duplicate tests (especially test names) which is not allowed.
                     //std::size_t,
+#if !defined(ALPAKA_CI)
                     std::int64_t,
+#endif
                     std::uint64_t,
                     std::int32_t,
+#if !defined(ALPAKA_CI)
                     std::uint32_t,
```

Comment:
I had to exclude the tests for `std::int64_t` and `std::uint32_t` as index type from the CI because the compilation times were much too high. While doing this I merged it with the special CUDA version from above.